guides/posts/build-car-data-app/post.md

```diff
@@ -0,0 +1,414 @@
+---
+title: How to build a Car Data app using Car Data API?
+description: Today, I am building a car data pp that will provide you with different car details depending on their make and model.
```

Comment:
Can you add the description of the app that you are building?

Reply:
[Done](https://github.com/RapidAPI/DevRel-Stack-Data/pull/121/commits/32dd1d6a793f1715d93020ed0ee101ab3e9e0a24).